src/content/docs/mobile-monitoring/new-relic-mobile-ios/get-started/introduction-new-relic-mobile-ios.mdx

```diff
@@ -24,24 +24,22 @@ redirects:
   - /docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation
 ---
 
-New Relic's mobile monitoring for iOS gives you a comprehensive view of your app's performance. It works for iOS apps written using Objective-C, Swift, or both languages. For a general overview of mobile monitoring functionality, see [Introduction to mobile monitoring](/docs/mobile-monitoring/new-relic-mobile/getting-started/introduction-new-relic-mobile).
+New Relic's mobile monitoring for iOS gives you a comprehensive view of your app's performance. It works for iOS apps written using Objective-C or Swift. For a general overview of mobile monitoring functionality, see [Introduction to mobile monitoring](/docs/mobile-monitoring/new-relic-mobile/getting-started/introduction-new-relic-mobile).
```

Comment:
```suggestion
Our iOS agent gives you a comprehensive view of your app's performance. The agent monitors iOS apps written in Objective-C, Swift, or both. For a general overview of mobile monitoring functionality, see [Introduction to mobile monitoring](/docs/mobile-monitoring/new-relic-mobile/getting-started/introduction-new-relic-mobile).
```

Reply:
I feel like all the mobile agent docs should match a similar naming schema (i.e., "Our iOS agent", or "The New Relic iOS agent"